tests/test_enterprise/api/test_views.py

```diff
@@ -4499,7 +4501,8 @@ def test_bulk_enrollment_in_bulk_courses_nonexisting_user_id(
     @mock.patch(
         'enterprise.api.v1.views.enterprise_customer.get_best_mode_from_course_key'
     )
-    @mock.patch("enterprise.utils.lms_enroll_user_in_course")
+    @mock.patch('enterprise.utils.lms_update_or_create_enrollment')
+    @override_settings(ENABLE_ENTERPRISE_BACKEND_EMET_AUTO_UPGRADE_ENROLLMENT_MODE=True)
```

Comment:
[suggestion] If there's a chance of having the feature flag disabled on production still, it might be worth having test coverage when `ENABLE_ENTERPRISE_BACKEND_EMET_AUTO_UPGRADE_ENROLLMENT_MODE ` is both `True` and `False` to assert both code paths still work separately. Otherwise, I believe we're losing coverage on the `else` path now?